app/Kernel.php

```diff
@@ -43,7 +43,7 @@ public function handle(Request $request, $type = HttpKernelInterface::MASTER_REQ
      * This will disappear in time in favour of container-driven parameters.
      * @deprecated
      */
-    protected function defineForkConstants()
+    public function defineForkConstants()
```

Comment:
I'm not really happy with this, but it's necessary to have these constants in our functional tests. They should be refactored/removed/replaced by DIC parameters in the future though.